### PR TITLE
rails4 master merge

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

File: lib/katello.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@
 require "rest_client"
 require "anemone"
 require "securerandom"
+require "strong_parameters"
```
Comment (cpeters) [inline lib/katello.rb]: I'm not sure we need this require statement.

Comment (komidore64) [inline lib/katello.rb]: seems it it related to [this issue](http://projects.theforeman.org/issues/12047), though i agree with you. in general we shouldn't need to manually require `strong_parameters`.

Comment (cpeters) [inline lib/katello.rb]: We don't have the strong_parameters gem declared in the katello.gemspec anymore so this require statement will break our rails4 branch

Comment (komidore64) [inline lib/katello.rb]: `strong_parameters` should already be included via our dependency on rails4

Comment (cpeters) [inline lib/katello.rb]: Has this been tested? I realize that all the tests won't pass when running the test suite but if the requires statement blows up we need to yank it out.

Comment (komidore64) [inline lib/katello.rb]: i was totally incorrect.

removing this `require`.

**Comment by cpeters** [comment]
ACK

**Pull request merged**